src/state_machine.cpp

```diff
@@ -202,3 +231,25 @@ State& StateMachine::createGaitState(const SectorAddress addr, const SectorAddre
 
   return normal;
 }
+
+State& StateMachine::createEscapeState(const SectorAddress address, const std::string& gait_name)
+{
+  this->escape_states_.emplace_back(address, gait_name);
+  ;
```

Comment:
:thinking:

Reply:
Hmm, I've had this before. Not really sure how/why this happens ... But I'll remove it 👍